ocr_flow.py

```diff
@@ -137,3 +57,44 @@ def upload_pdf():
         return jsonify({'gcs_pdf_url': gcs_pdf_url, 'file_id': str(file_id), 'ocr_result': ocr_results}), 200
     
     return jsonify({'gcs_pdf_url': "fake"}), 400
+
+def ocr_flow(uploaded_file, file_id):
```

Comment:
whats the diff between this and the old ocr_flow function you removed

Reply:
Same thing except it doesn't call `chunk_elements` and I moved it down in the file